### ISSUE TITLE
pip install -r fails on Ubuntu 16.04 w/ Python3

### ISSUE THREAD

**Issue by thisisjaid** [phageParser/phageParser#245]
It seems that pyscopg2 requires pg_config, otherwise fails to build. On Ubuntu 16.04 this can be solved by installing libpq-devel, packages/solutions for other OSes vary. We should add this to the install instructions.

There's also an additional error from python not being able to build wheels for everything (pip install wheel would solve that) but that's less important as it doesn't cause the installs to fail.

**Comment by thisisjaid** [comment]
Selenium seems to also be missing from the requirements and had issues with geckodriver missing as well when trying to use crisprfinder.py. We should take care of all of these ideally.

**Comment by FenghuJ** [comment]
Yes,I just install the phageParse and I use the firefox 52.7.3 instead of the newest one ,my system is Ubuntu 16.04,and it got into error that
`No such file or directory: 'geckodriver'`
`selenium.common.exceptions.WebDriverException: Message: 'geckodriver' executable needs to be in PATH`,so how can I fix this ? Can you give me some advice.
